index.html

```diff
@@ -2096,7 +2096,12 @@ <h2>Hypermedia Controls Vocabulary Definitions</h2>
           "rfc2119" title="MUST">MUST</em> include an entry in the array
           associated with the name <code>additionalResponses</code> that
           includes a value for the name <code>schema</code>.</span> 
-          </p></section>
+          </p><p>
+          A <a>Thing Description</a> of a WoT producer may has multiple forms entries with, e.g., different 
+          protocol and/or content types declarations that a <a>Consumer</a> would also support. In that case
```

Comment:
"could possibly support" (may be more clear to me, but you decide)

Reply:
done